pkg/apis/camel/v1alpha1/kamelet_types_support.go

```diff
@@ -192,13 +192,10 @@ func ValidKameletProperties(kamelet *Kamelet) bool {
 	return true
 }
 
-func ValidKameletTemplate(kamelet *Kamelet) bool {
+func ValidKamelet(kamelet *Kamelet) bool {
```

Comment:
Looks like this function doesn't do anything anymore. Probably better to remove it. Or add some meaningful check here.

Reply:
It just checks the kamelet is not nil. I guess we may need it in the future to provide more checks, but for now it really doesn't do that much. I may remove it at next iteration, thanks.